money_honey_cr.py

```diff
@@ -0,0 +1,147 @@
+        :param name: Name of the input.
+        :param currency: Currency symbol or letter code.
+        :param symbol_currencies: Array of currency codes for specific symbol.
+        :return: Check function convert_symbol for returns.
+        """
+        if symbol_currencies is not None and currency.upper() in symbol_currencies:
+            return currency
+        elif symbol_currencies is None and currency_changer.is_supported_currency(currency.upper()):
+            return self.convert_symbol(currency_changer, name, currency)
+        else:
+            print("Incorrect " + name[2:] + ". Either it isn't supported or you messed up on your part.")
+            sys.exit(1)
+
+    def convert_symbol(self, currency_changer, input_name, symbol):
+        """
+        Converts all symbols to currency code and puts "--output_currency" into array.
+        :param currency_changer: Instance of class actual_currencies.CurrenciesChanger.
+        :param input_name: Name of the input. [--input_currency, --output_currency]
+        :param symbol: Symbol or currency code.
+        :return: Value if input_name equals "--input_currency", array otherwise.
+        """
+        if symbol in list(currency_changer.currency_signs.keys()):
+            currency_codes = currency_changer.currency_signs[symbol]
+            if input_name == "--input_currency" and len(currency_codes) > 1:
+                if input("Would you care to specify? y/n\n").lower() in ['y', 'yes']:
+
+                    print("You can choose from: [", end='')
+                    for i in range(len(currency_codes)):
+                        if i == len(currency_codes) - 1:
+                            print(currency_codes[i] + "]")
+                        else:
+                            print(currency_codes[i] + "; ", end='')
+
+                    return self.check_currency(currency_changer, input_name, input("Enter specific currency code:\n"),
+                                               currency_codes).upper()
+                else:
+                    num_options = len(currency_codes)
+                    return currency_codes[randint(0, (num_options - 1))]
+            else:
+                return currency_codes[0] if input_name == "--input_currency"\
+                                            and len(currency_codes) == 1 else currency_codes
+        else:
+            # it's not symbol
+            return symbol.upper() if input_name == "--input_currency" else [symbol.upper()]
+
+    @staticmethod
+    def usage():
+        print("--amount\n\tRequired. Can contain only number.\n"
+              "--input_currency\n\tRequired. Can contain only three chars or currency symbol.\n"
+              "--output_currency\n\tOptional. Can contain only three chars or currency symbol.\n"
+              "--no_json\n\tOptional. Will change output format from json to custom one.")
+
+
+if __name__ == "__main__":
+    MoneyHoney().arguments_handler()
```

Comment:
as I understand this is main app.. 

this app can just handle args?

No conversion?

expect something like:
```
money_honey = MoneyHoney()
money_honey.arguments_handler()
money_honey.convert()
```